pkgs/applications/misc/kjv/default.nix

```diff
@@ -1,43 +1,41 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch }:
+{ lib, stdenv, fetchFromGitHub, fetchpatch, readline }:
 
 let
+  patchPrefix = "https://github.com/samuelgrf/kjv/commit/";
 
-patch-base = "https://github.com/LukeSmithxyz/kjv/commit/";
-
-add-apocrypha = fetchpatch {
-  url = patch-base + "b92b7622285d10464f9274f11e740bef90705bbc.patch";
-  sha256 = "0n4sj8p9m10fcair4msc129jxkkx5whqzhjbr5k4lfgp6nb1zk8k";
-};
-
-add-install-target = fetchpatch {
-  url = patch-base + "f4ad73539eb73f1890f4b791d8d38dd95900a4a4.patch";
-  sha256 = "1yzj72i5fkzn2i4wl09q6jx7nwn2h4jwm49fc23nxfwchzar9m1q";
-};
+  add-apocrypha = fetchpatch {
+    url = patchPrefix + "3f5cc5cbb7b5a94d5dd8b17a450fac4a3e162a32.patch";
+    sha256 = "sha256-5xxeem3Fg1C0L+zp7Ks/T+3dSXLjpXcM5morLFQYk74=";
+  };
 
+  add-install-target = fetchpatch {
+    url = patchPrefix + "87cef02980e9c3450855f8f3478d354bdf7d5960.patch";
+    sha256 = "sha256-+3uCP609+VRTDI3+PN6V/CV2APDGbqNMIrLJY1T1aS8=";
+  };
 in
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "kjv";
-  version = "unstable-2018-12-25";
+  version = "unstable-2021-03-11";
 
   src = fetchFromGitHub {
     owner = "bontibon";
-    repo = pname;
-    rev = "fda81a610e4be0e7c5cf242de655868762dda1d4";
-    sha256 = "1favfcjvd3pzz1ywwv3pbbxdg7v37s8vplgsz8ag016xqf5ykqqf";
+    repo = "kjv";
+    rev = "108595dcbb9bb12d40e0309f029b6fb3ccd81309";
+    hash = "sha256-Z6myd9Xn23pYizG+IZVDrP988pYU06QIcpqXtWTcPiw=";
   };
 
   patches = [ add-apocrypha add-install-target ];
 
-  makeFlags = [
-    "PREFIX=${placeholder "out"}"
-  ];
+  buildInputs = [ readline ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
```

Comment:
I recall being instructed to use `${placeholder "out"}` here.  Is there a reason to change it?

Reply:
I changed it simply because it's shorter.

But using `placeholder` is actually the better approach, because it doesn’t rely on variable expansion in Make.
See: https://discourse.nixos.org/t/generic-builder-minimal-autoconf-example/880/2

I reverted this change in the last force push.